Essentials/src/main/java/com/earth2me/essentials/signs/SignEnchant.java

```diff
@@ -20,8 +21,10 @@ public SignEnchant() {
     @Override
     protected boolean onSignCreate(final ISign sign, final User player, final String username, final IEssentials ess) throws SignException, ChargeException {
         final ItemStack stack;
+        final String itemName = sign.getLine(1);
+        final MaterialTagProvider tagProvider = ess.getMaterialTagProvider();
         try {
-            stack = sign.getLine(1).equals("*") || sign.getLine(1).equalsIgnoreCase("any") ? null : getItemStack(sign.getLine(1), 1, ess);
+            stack = itemName.equals("*") || itemName.equalsIgnoreCase("any") || tagProvider.tagExists(itemName) ? null : getItemStack(sign.getLine(1), 1, ess);
```

Comment:
As it stands this will blow up on 1.12 and below. We should either make this null-safe or add a no-op provider implementation.